action.yml

```diff
@@ -77,6 +77,11 @@ runs:
         npm_test_script_name: ${{ inputs.npm_test_script_name }}
         sonar_wait_flag: ${{ steps.sonar-setup.outputs.wait_flag }}
 
+    - name: Sonar Coverage Check
```

Comment:
can you add for dotnet as well?

Reply:
I did :)
https://github.com/variant-inc/actions-dotnet/pull/39